src/pages/register-a-beacon/primary-beacon-use.tsx

```diff
@@ -50,77 +81,121 @@ const PrimaryBeaconUse: FunctionComponent = (): JSX.Element => {
 };
 
 const BeaconUseForm: FunctionComponent<BeaconUseFormProps> = ({
-  heading,
-}: BeaconUseFormProps) => (
-  <Form action="/register-a-beacon/primary-beacon-use">
-    <FormFieldset>
-      <FormLegendPageHeading>{heading}</FormLegendPageHeading>
-    </FormFieldset>
-    <RadioListConditional>
-      <RadioListItemHint
-        id="motor-vessel"
-        name="maritimePleasureVesselUse"
-        value={MaritimePleasureVessel.MOTOR}
-        hintText="E.g. Speedboat, RIB"
-      >
-        Motor vessel
-      </RadioListItemHint>
-      <RadioListItemHint
-        id="sailing-vessel"
-        name="maritimePleasureVesselUse"
-        value={MaritimePleasureVessel.SAILING}
-        hintText="E.g. Skiff, Dinghy, Yacht, Catamaran"
-      >
-        Sailing vessel
-      </RadioListItemHint>
-      <RadioListItemHint
-        id="rowing-vessel"
-        name="maritimePleasureVesselUse"
-        value={MaritimePleasureVessel.ROWING}
-        hintText="E.g. Single person rowing boat, Cornish Gig, Multi-person rowing boat"
-      >
-        Rowing vessel
-      </RadioListItemHint>
-      <RadioListItemHint
-        id="small-unpowered-vessel"
-        name="maritimePleasureVesselUse"
-        value={MaritimePleasureVessel.SMALL_UNPOWERED}
-        hintText="E.g. Canoe, Kayak"
-      >
-        Small unpowered vessel
-      </RadioListItemHint>
-      <RadioListItemHint
-        id="other-pleasure-vessel"
-        name="maritimePleasureVesselUse"
-        value={MaritimePleasureVessel.OTHER}
-        hintText="E.g. Surfboard, Kitesurfing"
-        inputHtmlAttributes={{
-          "data-aria-controls": "conditional-other-pleasure-vessel",
-        }}
-      >
-        Other pleasure vessel
-      </RadioListItemHint>
-      <RadioListItemConditional id="conditional-other-pleasure-vessel">
-        <FormGroup>
-          <Input
-            id="other-pleasure-vessel-text"
-            name="otherPleasureVesselText"
-            label="What sort of vessel is it?"
-          />
-        </FormGroup>
-      </RadioListItemConditional>
-    </RadioListConditional>
-
-    <Button buttonText="Continue" />
-  </Form>
-);
-
-export const getServerSideProps: GetServerSideProps = withCookieRedirect(
-  async () => {
+  formData,
+  checkedValue = null,
+  showErrors,
+  errorMessages,
+}: BeaconUseFormProps): JSX.Element => {
+  const setCheckedIfUserSelected = (userSelectedValue, componentValue) => {
     return {
-      props: {},
+      defaultChecked: userSelectedValue === componentValue,
     };
-  }
+  };
+
+  return (
+    <Form action="/register-a-beacon/primary-beacon-use">
+      <FormGroup showErrors={showErrors}>
+        <FormFieldset>
+          <FormLegendPageHeading>
+            What type of maritime pleasure vessel will you mostly use this
+            beacon on?
+          </FormLegendPageHeading>
+        </FormFieldset>
+        {showErrors && <FieldErrorList errorMessages={errorMessages} />}
```

Comment:
If you merge in `main` branch take a look at the `FieldErrorList` component as you shouldn't have to do this conditional logic here.  Related to comment above

Reply:
The only prop in `FieldErrorList` in `main` is the `errorMessages` array.  There is no boolean `showErrors` equivalent.  Maybe it's on a WIP branch still?  I won't make `showErrors` a prop on this PR to avoid merge conflicts.

Reply:
Sorry my fault! It's an input to the `FormGroup` component if that helps?